src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java

```diff
@@ -403,6 +390,41 @@ public void afterNavigateTo(String url, WebDriver driver) {
     logJSError();
   }
 
+  /**
+   * All of tests should be executed as an user who opted in (agreed) on using ads tracking,
+   * unless you change it in @Execute annotation.
+   * Manually user would need to click 'agree' in the tracking opt in modal.
+   */
+  private void handleGdprModalWithCookie() {
+    driver = DriverProvider.getActiveDriver();
+    Method method = TestContext.getCurrentTestMethod();
+    if (TestContext.isFirstLoad()) {
+      boolean userOptedIn = true;
+      boolean userOptedOut = false;
+
+      if (method.isAnnotationPresent(Execute.class) && !method.getAnnotation(Execute.class)
+          .trackingOptIn()) {
+        userOptedIn = false;
+      }
+
+      if (method.isAnnotationPresent(Execute.class)) {
+        userOptedOut = method.getAnnotation(Execute.class).trackingOptOut();
+      }
+
+      if (userOptedIn) {
+        driver.manage().addCookie(
+            new Cookie("tracking-opt-in-status", "accepted", ".wikia.com", "/",
+                       new Date(new DateTime().plusYears(10).getMillis())));
+        driver.manage().addCookie(new Cookie("tracking-opt-in-status", "accepted"));
+      } else if (userOptedOut) {
+        driver.manage().addCookie(
+            new Cookie("tracking-opt-in-status", "rejectedgit", ".wikia.com", "/",
+                       new Date(new DateTime().plusYears(10).getMillis())));
+      }
+//      driver.navigate().refresh();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3ACommentedOutCodeLine)